bin/dcps_tests.lst

```diff
@@ -409,3 +409,9 @@ tests/DCPS/RtpsRelay/Smoke/run_test.pl single: !DCPS_MIN CXX11 RTPS !NO_BUILT_IN
 tests/DCPS/RtpsRelay/STUN/run_test.pl: !DCPS_MIN CXX11 RTPS !NO_BUILT_IN_TOPICS !OPENDDS_SAFETY_PROFILE !IPV6
 
 tests/DCPS/ParticipantLocationTopic/run_test.pl noice: !DCPS_MIN CXX11 RTPS !NO_BUILT_IN_TOPICS !OPENDDS_SAFETY_PROFILE !IPV6
+
+performance-tests/bench_2/run_test.pl disco: !DCPS_MIN !NO_MCAST RTPS !DDS_NO_OWNERSHIP_PROFILE CXX11 RTPS
```

Comment:
```suggestion
performance-tests/bench_2/run_test.pl disco: !DCPS_MIN !NO_MCAST RTPS !DDS_NO_OWNERSHIP_PROFILE CXX11
```

Reply:
similar idea on a few lines below...

Reply:
They're getting run. The fan_frag test fails sometimes on the PR builder, which we'll want to look into, but I don't think it's the test's fault so I'm OK with that for now. I'm also curious to see if there are any platform / feature specific failures that we'll want to focus on first (from the scoreboard), which is why I haven't done a deep dive there yet. Just trying to get the tests in place first.